### PR TITLE
Feature/addattachment

### PR DESCRIPTION
**Pull request by derkgort**
Added option to upload/add an attachment to an SalesInvoice.
This is mostly heavily inspired by the change suggested by @dannyvw in #102. New issue: #157

### CODE REVIEW

File: src/Picqer/Financials/Moneybird/Entities/SalesInvoice.php [hunk 1]
```diff
@@ -200,4 +200,27 @@ public function duplicateToCreditInvoice()
...
+     *
+     * @throws \Picqer\Financials\Moneybird\Exceptions\ApiException
+     */
+    public function addAttachment($file)
+    {
+        $this->connection()->upload($this->endpoint . '/' . $this->id . '/attachments', [
+            'multipart' => [
+                [
+                    'name' => 'file',
+                    'contents' => fopen($file, 'r'),
```
Comment (stephangroen) [inline src/Picqer/Financials/Moneybird/Entities/SalesInvoice.php]: I think we should just pass the file contents to the `addAttachment` method. This way anyone who implements it has their own choice of how to get the contents. For example if someone has contents in a database, they would query it and pass it to the method.

Comment (derkgort) [reply src/Picqer/Financials/Moneybird/Entities/SalesInvoice.php]: Sounds reasonable. I will change it to do that.

Comment (stephangroen) [reply src/Picqer/Financials/Moneybird/Entities/SalesInvoice.php]: Thanks!

**Comment by gizburdt** [comment]
Would love this feature! Can this be merged? :)

**Comment by stephangroen** [comment]
Thanks for the change @derkgort :)

**Pull request merged**